### ISSUE TITLE
Triton group normalization is slower than torch ops in some case.

### ISSUE THREAD

**Issue by chenly15** [chengzeyi/stable-fast#134]
In the techniques, I found GroupNorm was effective to speed up. But when I tested the speed between triton and torch operations, I found triton was slower than torch. The script is the same as [https://github.com/chengzeyi/stable-fast/blob/main/src/sfast/triton/ops/group_norm.py](url), and the result is shown below. 
For the GroupNorm operation, whether triton is faster than torch. Can I replace the torch.nn.GroupNorm with TritonGroupNorm directly, to accelerate stable diffusion model. 
<img width="434" alt="截屏2024-03-12 17 51 00" src="https://github.com/chengzeyi/stable-fast/assets/33417982/cc92c652-1f78-4282-9b43-a941010a1b21">
My env is
A100 GPU, torch 2.1, triton 2.1, no xformers, diffusers 0.21.2

**Comment by chengzeyi** [comment]
@chenly15 Our implementation may be not super effecient now. I currently use other methods to speed up groupnorm computation. However, it has not been open-sourced by far.
